standard/recommendations/core/PER_links.adoc

```diff
@@ -2,7 +2,7 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Permission {counter:per-id}* |*/per/core/links*
-^|A |A WNM `+links+` array property MAY provide link objects which reference APIs or Web Accessible Folders (WAF).
+^|A |A WNM `+links+` array property MAY provide link objects which reference APIs or web-accessible folders (WAF).
```

Comment:
Web Accessible Folders (WAF) is a known term in spatial data infrastructure (much like "Web Map Service" not "Web map service").